packages/server/__tests__/isTeamActive.test.ts

```diff
@@ -0,0 +1,195 @@
+import {ThresholdTest as Threshold} from '~/types/constEnums'
+import isTeamActive from '../graphql/public/types/helpers/isTeamActive'
+import getKysely from '../postgres/getKysely'
+
+// Mock data
+const mockOrgId = 'org123'
+const mockTeamId = 'team123'
+const mockUserId1 = 'user1'
+const mockUserId2 = 'user2'
+
+describe('isTeamActive', () => {
+  const pg = getKysely()
+
+  const cleanupTestData = async () => {
+    await pg.deleteFrom('NewMeeting').where('teamId', '=', mockTeamId).execute()
```

Comment:
+1 There are also https://github.com/ParabolInc/parabol/blob/4aee869e4c3f803e4917d45c41b3b04e1f81fe8b/packages/server/__tests__/common.ts#L238 and https://github.com/ParabolInc/parabol/blob/4aee869e4c3f803e4917d45c41b3b04e1f81fe8b/packages/server/__tests__/common.ts#L251 see for example the `isOrgVerified` tests https://github.com/ParabolInc/parabol/blob/4aee869e4c3f803e4917d45c41b3b04e1f81fe8b/packages/server/dataloader/__tests__/isOrgVerified.test.ts#L54